.travis.yml

```diff
@@ -52,22 +52,51 @@ before_script:
 jobs:
   include:
     - stage: test
+      name: "Code check & linting"
       before_script:
         - aio/scripts/install-codegen.sh
       script: npm run check
-    - script: npm run test:coverage
+    - name: "Unit tests"
+      script: npm run test:coverage
       after_success:
         - rm -rf $TRAVIS_BUILD_DIR/.tmp
         - bash <(curl -s https://codecov.io/bash)
-    - script: npm run cluster:start && npm run e2e
+    - name: "Helm linting"
+      cache: false
+      install:
+        - curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+        - wget https://github.com/garethr/kubeval/releases/download/0.14.0/kubeval-linux-amd64.tar.gz
+        - mkdir kubeval
+        - tar xf kubeval-linux-amd64.tar.gz -C kubeval
+        - export PATH="$PATH:$(pwd)/kubeval"
+      before_script: skip  # We don't need Docker nor Go
+      script:
+        - aio/scripts/helm-lint.sh
+    - name: "e2e tests"
+      script: npm run cluster:start && npm run e2e
+
     - stage: deploy
+      name: "Development release"
       script:
         - docker login -u $DOCKER_USER -p $DOCKER_PASS
         - npm run docker:push:head
+
     - stage: release
+      name: "Release"
       script:
         - docker login -u $DOCKER_RELEASE_USER -p $DOCKER_RELEASE_PASS
         - npm run docker:push
+    # Manual step for now since it requires to add a GitHub Token to Travis which would be a security issue
```

Comment:
We have already tokens for Docker, Sauce etc. There is even already `GH_TOKEN` you can try if this is valid token already. If not we can add something that will enable this step :)

Reply:
Nice!
However I propose we merge this without change, manually release using the current procedure, then automate in a follow-up PR, what do you think?

Reply:
Ok, sure.